install/installing-custom-components.md

```diff
@@ -0,0 +1,198 @@
+# Installing Individual Knative Components
```

Comment:
good point. I agree with changing out components. I was trying to keep the model of filenaming that uses gerunds to represent a Task type topic, especially since everything is under a single directory (ie. so there is some way to identify the topics in folder w/o opening each file). 

This also has impact on SEO. Plus, I'd like to avoid qualifying the files with Knative since thats already in the URL path (and is pain if the name ever has to change).

installing-custom-packages?

Reply:
I agree on the parallelism and the SEO points, but I also want to make sure the topic name isn't misleading. installing-custom-packages makes me think of installing outside customizations that aren't a part of Knative...

Some ideas:
performing-custom-installation
customizing-knative-installation (still has the "Knative," but at least it starts with a gerund...)

Reply:
per chat, renamed file to align with existing files. a separate effort (later) will revise and improve filenames